conf/mesos-site.xml

```diff
@@ -89,6 +89,21 @@
     <value>3</value>
   </property>
 
+  <property>
+    <name>mesos.hdfs.jre-url</name>
+    <value>https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz</value>
+  </property>
```

Comment:
Let's remove the additions to this file since the defaults are already set in HDFSFrameworkConf now